docs/reference/tab-widgets/troubleshooting/troubleshooting-shards-capacity.asciidoc

```diff
@@ -0,0 +1,208 @@
+
+// tag::non-frozen-nodes-cloud[]
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment.
++
+
+NOTE: If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support].
+If your deployment doesn't include {kib}, all you need to do is
+{cloud}/ec-access-kibana.html[enable it first].
+
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Dev Tools > Console**.
++
+[role="screenshot"]
+image::images/kibana-console.png[{kib} Console,align="center"]
+
+. Update the <<cluster-max-shards-per-node,`cluster.max_shards_per_node`>> setting:
```

Comment:
I think in all cases we should as a first step inspect the current value of the setting. 
Either via the `GET _cluster/settings` API or via the `_health_report/shards_capacity` API. 

But let's please display that step and indicate that we need to increase the value of the setting relative to what's currently configured  - here's an example of how we indicate this for a cluster setting https://elasticsearch_95208.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/increase-cluster-shard-limit.html